ui/views/routing.go

```diff
@@ -0,0 +1,464 @@
+package views
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/jroimartin/gocui"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+
+	"github.com/edouardparis/lntop/config"
+	netmodels "github.com/edouardparis/lntop/network/models"
+	"github.com/edouardparis/lntop/ui/color"
+	"github.com/edouardparis/lntop/ui/models"
+)
+
+const (
+	ROUTING         = "routing"
+	ROUTING_COLUMNS = "routing_columns"
+	ROUTING_FOOTER  = "routing_footer"
+)
+
+var DefaultRoutingColumns = []string{
+	"DIR",
+	"STATUS",
+	"IN CHANNEL",
+	"IN ALIAS",
+	"OUT CHANNEL",
+	"OUT ALIAS",
```

Comment:
sorry it is missing `_`
Columns are not displayed

Reply:
Oops, fixed in f0dedd5.